include/ceed/backend.h

```diff
@@ -66,14 +66,18 @@
 
 /// CEED_DEBUG_COLOR default value, forward CeedDebug* declarations & macros
 #ifndef CEED_DEBUG_COLOR
-#define CEED_DEBUG_COLOR 0
+#define CEED_DEBUG_COLOR 255
```

Comment:
Hard-coding this color is not my idea of a good time.
![image](https://user-images.githubusercontent.com/3303/141364574-040a87d5-e357-4c10-b49e-9f277ce6df01.png)

Reply:
Fixed. I meant to use that as a flag to use the env default since its not a super common color.